tests/tom-swifty-test/SwiftReflector/BindingImportTests.cs

```diff
@@ -23,7 +23,7 @@ public void LoadsDatabaseSmokeTest (PlatformName platform, int expectedLowerLimi
 			var database = importer.Import ();
 			Assert.IsNotNull (database, $"null database for {platform}");
 			Assert.Less (expectedLowerLimit, database.Count, $"Expected at least {expectedLowerLimit} db entries, but got {database.Count} entries.");
-			Assert.IsTrue (!errors.AnyErrors, $"{errors.ErrorCount} errors importing database.");
+			errors.AssertNoErrors ("importing database.");
```

Comment:
Sanity check - would it make sense to make the "importing database" a constant for these tests?

Reply:
Sure! PRs welcome! 😄